.studio/ingest-service

```diff
@@ -14,6 +14,7 @@ function ingest_integration() {
   local rc
 
   # Go based integration tests
+  export PG_URL="postgresql://ingest@127.0.0.1:10145/chef_ingest_service?sslmode=verify-ca&sslcert=/hab/svc/ingest-service/config/service.crt&sslkey=/hab/svc/ingest-service/config/service.key&sslrootcert=/hab/svc/ingest-service/config/root_ca.crt"
```

Comment:
Alternatively I did this trick in the applications service to get the PG URL: https://github.com/chef/automate/blob/master/.studio/applications-service#L158

Reply:
And in the integration test: https://github.com/chef/automate/blob/master/components/applications-service/integration_test/suite_test.go#L47

Both did the trick! :100: